### ISSUE TITLE
Quick Start Broken?

### ISSUE THREAD

**Issue by martyall** [purenix-org/purenix#52]
Hey, I was trying to follow the [Quick Start](https://github.com/purenix-org/purenix/blob/main/docs/quick-start.md) but couldn't get the example to build. After following the steps, including switching out the package set hash to the latest on  `master` of the `temp-package-set` repo. Running `spago build` after that complained that that package set required an older version of `purs` (0.14.something), but I chose the option to override it in the `packages.dhall` file..

The result of running `spago build` was that modules compiled to `CoreFn` but `purenix` failed:

``` bash
[info] Build succeeded.
purenix: impossible
CallStack (from HasCallStack):
  error, called at src/PureNix/Identifiers.hs:63:30 in purenix-1.1-AKRNnyJSalY6yydb60EpDF:PureNix.Identifiers
[error] Backend "purenix" exited with error:1
```

Are you able to reproduce this? FWIW I also tried to build a custom nix shell using EasyPurescriptNix with the appropriate version of purs that spago wanted for that package set, and I still got errors, albeit a different one. Something about not finding `sourcePos`. 

This is my first rodeo trying to use a different PureScript backend, and I'm not a nix power user so maybe I'm just missing something :thinking:

**Comment by jonascarpay** [comment]
Thanks for opening this issue! It's definitely not you, turns out we were a bit premature merging the PureScript 0.15 support.

I fixed the issue in #53, would you mind giving that a shot? You should be able to drop into a shell with the hotfix using:
```
nix develop github:purenix-org/purenix/fe08f5b9384b1a13649239ab6d783f94efcd3a3d#use-purenix
```

**Comment by martyall** [comment]
yup, it works!
